C++/libs/valgrind.txt

```diff
@@ -0,0 +1,172 @@
+==36284== Memcheck, a memory error detector
```

Comment:
Why is this file here? Is this configuration for valgrind?

Reply:
That was also an accidental artifact left over from a debugging session.  It has been removed.